.github/workflows/auto-pr.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto PR for new versions
+
+on:
+  schedule:
+  - cron: 0 0 * * *
```

Comment:
add a comment about what this means so I don't have to use https://crontab.guru/?